know-your-computer/readme.md

```diff
@@ -0,0 +1,34 @@
+# Know Your Computer
+
+"Know Your Computer" focuses on how to perform regular tasks on your computer. In this workshop you will use basic Linux commands in the terminal. The goal is to reduce your fear of using the terminal. 
+
+## Pre-requisite
+To be able to do this workshop you will need a computer with a Linux shell running on it. If you already have a Mac or an Ubuntu operating system or any other Linux distribution running on your laptop, then you are fine. If your laptop is running on Windows operating system, download and install [Git Bash for Windows](https://git-scm.com/downloads).
+Code Your Future does not support Windows, so you must install or dual boot Linux to take our course. But to do this workshop, you can just use Git Bash.
+Once installed, open git bash and you should be able to perform the tasks below. 
+
+## Tasks
+You must use the terminal and Linux commands to complete each challenge. You must not use your GUI! Hands off the mouse! 🖱️ 
+
+ - Create a folder called my-shopping inside your Documents folder.
```

Comment:
Format the filenames to make them stand out from the text, whatever the format standard is for the workshops repository (eg backticks or bold text)